ui/helpers/constants.js

```diff
@@ -213,6 +213,8 @@ export const thEvents = {
   openLogviewer: 'open-logviewer-EVT',
   autoclassifyIgnore: 'ac-ignore-EVT',
   applyNewJobs: 'apply-new-jobs-EVT',
+  filtersUpdated: 'filters-updated-EVT',
+  clearPinboard: 'clear-pinboard-EVT',
```

Comment:
I also needed ``clearPinboard`` to keep functionality the way it was prior to this where changing range would reload the page.  So the pinboard would get cleared.  Otherwise, you could have pinned jobs from a push that is no longer in range and classify the wrong job.  When I convert the ``pinnedJobs`` context, I may make this a little smarter (prune to the pinned jobs that are still in range)